### ISSUE TITLE
Element IDs are not unique (featured content on Archivo home page)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5345]
Sites identified: Archivo
Page(s) identified: home page (#featured_content ID is repeated in translation of content)

WCAG compliance level: A

Description:

Two or more elements on this page have the same id value. These should be unique.

Sometimes it's important to know how different page elements fit together.

Whereas sighted visitors may be able to get this information from the visual layout, visitors using screen readers rely on page elements being marked up correctly in the HTML.

See more details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1166400/A11Y/Page?pageId=37942903524&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r3/failed/dnpERzNIYUY=;Mjg1M2VhYjg=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcmhnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--1256a088c7a9ba410d6eb5535338799a7f28804f/image.png)

**Comment by cjcolvar** [comment]
Fix deployed to archivo-staging and ready for review.
